widget/ui/package.json

```diff
@@ -26,7 +26,6 @@
   },
   "peerDependencies": {
     "@lingui/core": "4.2.1",
-    "@lingui/react": "4.2.1",
```

Comment:
What was the need to remove `@lingui/react` from `peerDependencies` and `devDependencies`
And we put it in `dependencies`?

Reply:
fixed it